install/docker/centos-docker-install.md

```diff
@@ -37,7 +37,13 @@
 
    2. Docker `daemon.json`
 
-       Add the following definitions to `/etc/docker/daemon.json`:
+      Create docker configuration folder.
+
+      ```bash
+      mkdir -p /etc/docker
```

Comment:
Probably need to prefix the command with `$`, as per the rest of the document, which also fits in with our docs guide at https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md

Reply:
Fixed.